client/my-sites/stats/stats-date-control/stats-date-control-picker-date.tsx

```diff
@@ -10,11 +11,16 @@ const DateControlPickerDate = ( {
 	onApply,
 	onCancel,
 }: DateControlPickerDateProps ) => {
+	const translate = useTranslate();
 	// TODO: Rename component?
 	// Feels a bit confusing now. Should have a better idea
 	// of appropriate names once hierarchy is finalized.
 	return (
 		<div className="date-control-picker-date">
+			<h2 className="date-control-picker-date__heading">
+				{ translate( 'Date Range' ) }
+				<span>(dd/mm/yyyy)</span>
```

Comment:
Note: We need to dynamically show user's format here.

Reply:
correct. I already created an issue for this in https://github.com/Automattic/wp-calypso/issues/83148